BootloaderCorePkg/Library/FspApiLib/FspmApiLib.inf

```diff
@@ -56,4 +56,4 @@
   gPlatformModuleTokenSpaceGuid.PcdFspResetStatus
 
 [FixedPcd]
-  gPlatformModuleTokenSpaceGuid.PcdFSPMUpdSize
```

Comment:
Remove it from DEC too?

Reply:
We could, but I thought leaving them might be useful to an SBL developer down the road.